lib/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module com.auth0.jwt {
+    requires com.fasterxml.jackson.databind;
+
+    exports com.auth0.jwt;
+    exports com.auth0.jwt.algorithms;
+    exports com.auth0.jwt.exceptions;
+    exports com.auth0.jwt.impl;
```

Comment:
Maybe for the next major release the export for `impl` should be removed? See #487
What do you maintainers think about this?

Reply:
Removing it from opens will completely hide it from library users, it will break code if somebody depends on it